src/blocks/remote-data-container/components/BlockBindingControls.tsx

```diff
@@ -161,6 +162,20 @@ export function BlockBindingControls( props: BlockBindingControlsProps ) {
 					/>
 				</>
 			);
+
+		case 'remote-data-blocks/remote-html':
+			return (
+				<>
+					<BlockBindingFieldControl
+						availableBindings={ availableBindings }
+						fieldTypes={ HTML_FIELD_TYPES }
+						label="Content"
+						target="content"
+						updateFieldBinding={ updateFieldBinding }
+						value={ contentField }
+					/>
+				</>
+			);
```

Comment:
Something we could consider in the future is making this filter-driven so plugin users can also reuse our remote data binding sidebar. Right now, we need to add supported blocks to this section of code.